### ISSUE TITLE
Not working for Sublime Text 3

### ISSUE THREAD

**Issue by mrtnbroder** [marijnh/tern_for_sublime#22]
Installed this via Packages Control ( Also tried manually ) but it always says "It appears Tern has not been installed...".

I think it's because how ST3 handles packages now.

**Comment by mladenuzelac** [comment]
I have the same problem with Sublime text 3 and windows 7 x64

**Comment by SAnDAnGE** [comment]
@mladenuzelac add (adjusting path) `"tern_command":["node", "C:/Users/blabla/AppData/Roaming/npm/node_modules/tern/bin/tern", "--no-port-file"]` to your Preferences -> Settings - User file.

**Comment by Whyser** [comment]
Have/had the same problem as @mladenuzelac did what @SAnDAnGE said and it "worked". Now I get no errors and it _seems_ that Tern starts. But it won't auto-complete anything.

It says: Starting TernJS server for empty with 1 libs and 0 files.

**Comment by OliverJAsh** [comment]
Is it worth adding a step for Mac users to install `https://github.com/int3h/SublimeFixMacPath` in the README? It's just caught me out again, reinstalling.

**Comment by marijnh** [comment]
Attached patch adds a short note about this.

**Comment by ourmaninamsterdam** [comment]
:+1: SublimeFixMacPath fixes it for me. Cheers

**Comment by pretodor** [comment]
Still having trouble making autocomplete work (I'm using Linux Mint 17.2 Cinnamon and ST3 Build 3114).

I tried installing it both manually and via the Package Manager. I added `"auto_complete_triggers": [ {"selector": "text.html", "characters": "<"}, {"selector": "source.js", "characters": "."} ]` to my Preferences file. I set:
```
"tern_argument_hints",
"tern_argument_completion",
"auto_complete"
```
to `true`. I followed @wtw's [advice](https://github.com/ternjs/tern_for_sublime/issues/22#issuecomment-23930109) by adding a `.tern-port` file all to no avail.

I also supposedly cleaned npm's cache and then updated both npm and minimatch. However, after that when I tried to install tern, I was still getting the following warnings:
```
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
```
Maybe the problem stems from there?
